pom.xml

```diff
@@ -4,19 +4,18 @@
     <parent>
         <groupId>org.jenkins-ci.plugins</groupId>
         <artifactId>plugin</artifactId>
-        <version>4.40</version>
+        <version>4.82</version>
         <relativePath />
     </parent>
     <groupId>io.jenkins.plugins</groupId>
     <artifactId>zohosprints</artifactId>
-    <version>2.1-SNAPSHOT</version>
+    <version>${revision}${changelist}</version>
     <packaging>hpi</packaging>
     <name>Zoho Sprints</name>
-    <description>Plugin which wil help to do action from jenkins while running the job</description>
```

Comment:
This has been inlined into index.jelly, because setting this in the plugin pom has no effect and will error in future versions.